akka-stream/src/main/scala/akka/stream/impl/fusing/GraphInterpreter.scala

```diff
@@ -296,8 +299,7 @@ final class GraphInterpreter(
         logic.preStart()
       } catch {
         case NonFatal(e) ⇒
-          e.printStackTrace()
```

Comment:
good, I noticed on more `printStackTrace` in `tryAbort` the other day, should that really be a `printStackTrace`